_publications/turilli2018building

```diff
@@ -0,0 +1,14 @@
+M. Turilli, A. Merzky, V. Balasubramanian, S. Jha. “Building Blocks for Workflow Sys-tem Middleware”. Accepted in Proceedings of the 18th IEEE/ACM International Sym-posium on Cluster, Cloud and Grid Computing (CCGrid), 2018.
+
+---
+title: "Building Blocks for Workflow System Middleware"
+collection: publications
+permalink: /publications/turilli2018building
+type: pub
+date: 2018-07-16
+author: "M. Turilli, A. Merzky, V. Balasubramanian, S. Jha"
+venue: "Proceedings of the 18th IEEE/ACM International Symposium on Cluster, Cloud and Grid Computing (CCGrid)"
+paperurl: https://ieeexplore.ieee.org/document/8411040
+---
+
```

Comment:
Don't you want to copy and paste the abstract here?

Reply:
Done